docs/user_guide/mac_install.md

```diff
@@ -56,7 +56,9 @@ Now install the [CDF](#cdf)
 
 You will also need the CDF (Common Data Format) library which can be downloaded from NASA. **Make sure you successfully installed the ncurses library for your distribution first.** 
 You can find the latest release at: [http://cdf.gsfc.nasa.gov/](http://cdf.gsfc.nasa.gov)
-For macOS it is also available through macports, as are all listed dependencies   
+For macOS it is also available through MacPorts, as are all listed dependencies. However, if the MacPorts installation errors or you do not use MacPorts, you may follow the instructions below.
+
+From the above site, navigate to the `macosx` directory and download the pre-compiled binary file `cdf[XX_X]-setup_universal_binary.tar.gz`, where `XX_X` is the version number of the latest release. Unpack the binary using `tar -xzvf cdf[XX_X]-setup_universal_binary.tar.gz` and double-click on the unpacked binary to be led through installation instructions.  Alternatively, you may follow the instructions given on the linux page, replacing the `make OS=linux ENV=gnu all` with `make OS=macosx ENV=gnu all`.  You can now delete the `cdfXX_X-dist` directory or binary and the `tar.gz` archive.
```

Comment:
Thanks @aburrell!! I just added the link to the linux install page...

```suggestion
From the above site, navigate to the `macosx` directory and download the pre-compiled binary file `cdf[XX_X]-setup_universal_binary.tar.gz`, where `XX_X` is the version number of the latest release. Unpack the binary using `tar -xzvf cdf[XX_X]-setup_universal_binary.tar.gz` and double-click on the unpacked binary to be led through installation instructions.  Alternatively, you may follow the instructions given on the [linux page](linux_install.md), replacing the `make OS=linux ENV=gnu all` with `make OS=macosx ENV=gnu all`.  You can now delete the `cdfXX_X-dist` directory or binary and the `tar.gz` archive.
```

Reply:
Thanks @ecbland, I couldn't recall how to do that!